components/reddit/common.js

```diff
@@ -0,0 +1,32 @@
+const reddit = require("./reddit.app");
+const get = require("lodash.get");
```

Comment:
I missed this last time, but I think we're supposed to include lodash packages like this:
`const get = require("loadsh/get");`
https://lodash.com/per-method-packages

Reply:
Ok noted, I commited the suggested change by jay